app/Services/UserService.php

```diff
@@ -9,27 +9,28 @@ class UserService
 {
     public function getUser($user)
     {
-        return User::where('id', $user)->first();
+        return User::find($user);
     }
 
-    public function updateUser($request, $profile)
+    public function updateUser($request, $id)
```

Comment:
Rename the parameter to `$data`, instead of `$request`. Update in all the methods in Service classes.